stories/tabs.stories.mdx

```diff
@@ -0,0 +1,46 @@
+import { Meta } from "@storybook/addon-docs/blocks";
+
+<Meta title="Next Version Implementation Guides/Components/Navigation/Tabs" />
+
+
+## Implementation guidelines for next tabs component 
+
+It should use text basic  component to display the labels 
```

Comment:
TextBasic
color TextPrimary for Active tab
color TextSecondary for Non-active tabs